JavaScript development/Frameworks/React.md

```diff
@@ -0,0 +1,489 @@
+# React Guidelines and Best Practices
+
+*Guides are not rules and should not be followed blindly. Use your head and think.*
+
+## File Organization and Naming
+- as it is a good practice to divide React components as container and presentational components, you should place them in separate folders (example: **containers** and **components**)
+
+```
+...
+├─ components
+├─ containers 
```

Comment:
in some cases this can be pages

Reply:
I commented on that above this example in the guide, and updated it to be more specific.